explore/src/clue/scala/queries/common/CategorizedTimeSubquery.scala

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package queries.common
+
+import clue.GraphQLSubquery
+import clue.annotation.GraphQL
+import lucuma.core.model.sequence.CategorizedTime
+import lucuma.schemas.ObservationDB
+import lucuma.odb.json.timeaccounting.given
+
+@GraphQL
+object CategorizedTimeSubquery
+    extends GraphQLSubquery.Typed[ObservationDB, CategorizedTime]("CategorizedTime"):
+  override val subquery: String = """
+    {
+      program {
```

Comment:
Is there any benefit to fetching all values other than being able to use the model case class? Since we never (thusfar at least) want to show any time other than the program time, I'd suggest only fetching that data, to prevent confusion

Reply:
@hugo-vrijswijk OK. I am now only pulling down program times, and I made it quite explicit everywhere.